include/aspect/utilities.h

```diff
@@ -1217,6 +1217,60 @@ namespace aspect
       });
       return sorted_vec;
     }
+
+    /**
+     * Contains utility functions related to tensors.
+     */
+    namespace Tensors
+    {
+
+      /**
+       * Rotate a 3D 4th order tensor with an other 3D 2th order tensor
+       */
+      SymmetricTensor<4,3>
+      rotate_4th_order_tensor(const SymmetricTensor<4,3> &input_tensor, const Tensor<2,3> &rotation_tensor);
```

Comment:
I'm reading this as "apply this rotation to that object", which would suggest to reverse the order of arguments.